### PR TITLE
fix tables, fix links, fix blocks

### PR DESCRIPTION
**Pull request by PSeitz**
- fix tables in index config
- fix notion links
- fix caution, tip block
- fix caution blocks

### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

File: docs/reference/storage-uri.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ The following are valid local file system URIs
 - ./quickwit
 ```
 
-<aside>
-⚠️ When using the `file://` protocol, a third `/` is necessary to express an absolute path.
+:::caution
```
Comment (fmassot) [inline docs/reference/storage-uri.md:33]: I think @fulmicoton add some dedicated css for <aside>, not sure if he wants to remove that. It's ok to keep the aside on a CSS point of view. Or did you see some issues?

Comment (PSeitz) [reply docs/reference/storage-uri.md:33]: yes, the rendering is broken with <aside>

Comment (fulmicoton) [reply docs/reference/storage-uri.md:33]: Oh no I just did not know about :::caution. Does it look good?

Comment (fulmicoton) [reply docs/reference/storage-uri.md:33]: arf no :::caution is broken.

**Review by fmassot — APPROVED** [review]
LGTM

**Review by fulmicoton — APPROVED** [review]

**Pull request merged**